devops/docker/smqtk_iqr_playground/README.md

```diff
@@ -15,8 +15,7 @@ the IQR GUI web-application as well as the underlying IQR RESTful service
 server.
 
 
-Quick Information
------------------
+### Quick Information
```

Comment:
I believe this should be `##` since `--` is [for subsections in sphinx](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections).